tools/wordcloud/macros.xml

```diff
@@ -0,0 +1,8 @@
+<macros>
+    <token name="@TOOL_VERSION@">1.9.4</token>
```

Comment:
If its just one tool, you can move this into the tool if you like - not as a separate file.

Reply:
Thanks for your comment. The files were updated based on your suggestion.

> If its just one tool, you can move this into the tool if you like - not as a separate file.

Thanks for your comment. The files were updated based on your suggestion.

Reply:
> If its just one tool, you can move this into the tool if you like - not as a separate file.

Thanks for your comment. The files were updated based on your suggestion.

Reply:
Ah, you misunderstood: using the tokens instead of hard-coded versions is important for automated update suggestions, but you can set the macros right in the tool xml instead of expanding them from another file.

Reply:
> Ah, you misunderstood: using the tokens instead of hard-coded versions is important for automated update suggestions, but you can set the macros right in the tool xml instead of expanding them from another file.

Thanks for your comment. I updated the file.